package.json

```diff
@@ -82,13 +82,14 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-react-perf": "^3.3.1",
     "generate-license-file": "^1.3.0",
-    "jest": "^27.5.1",
+    "jest": "^29.1.1",
+    "jest-environment-jsdom": "^29.1.1",
```

Comment:
to confirm, we still need to install this separately for v29?

Reply:
yes, for v28 and higher (including v29) `jest-environment-jsdom` must be installed separately